src/maps/mod.rs

```diff
@@ -478,7 +475,7 @@ impl BpfHashMap {
     /// the `BpfHashMap` behavior is undefined.
     ///
     /// # Examples
-    /// ```
+    /// ```no_run
```

Comment:
I'm not hugely concerned because these are doctests, though we should try to fix them. It's weird that these are failing while the other tests pass, that ostensibly exercise the same code.